examples/README.md

```diff
@@ -1,157 +1,55 @@
-# Instrumenting Example Applications with the OpenTelemetry .NET Automatic Instrumentation
+# Demonstrative example
 
-The following examples show how different instrumentation scenarios are covered by
-the OpenTelemetry .NET Automatic Instrumentation.
+## Description
 
-## Overview
+This example uses Docker Compose.
+It consists of following services:
 
-The example script instruments two .NET processes. These processes are identified
-as [OpenTelemetry services](https://github.com/open-telemetry/opentelemetry-specification/blob/d6bcc0cb072d8d6f6ced856f1f23c451648a3caa/specification/resource/semantic_conventions/README.md#service)
-with the following names:
+1. [`cli`](cli) - console application that makes a HTTP GET request
+2. [`srv`](srv) - simple HTTP server using SQL Server
+3. `sqlserver` - [Microsoft SQL Server](https://hub.docker.com/_/microsoft-mssql-server)
+   used by `srv`
+4. `otel-collector` - [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/)
+   which collects the telemetry send by `cli` and `srv`
+5. `jaeger` - [Jaeger](https://www.jaegertracing.io/) as traces backend
+6. `prometheus` - [Prometheus](https://prometheus.io/) as metrics backend
 
- 1. **`aspnet-server`**: ASP.NET Core application that makes requests to MongoDB
- and Redis databases.
- 2. **`{exampleApp}`**: .NET application that makes a few HTTP requests
- to the `aspnet-server` and some public websites. The exact name is equal
- to `exampleApp` value.
+## Usage
 
-The script creates Docker containers for MongoDB, Redis, Jaeger, Prometheus,
-and the [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/).
-You can see the traces generated by the example in the Jaeger container,
-while the metrics are available on the Prometheus container.
-The number and organization of traces generated depends on the `exampleApp`
-used in the specific run. The script waits for your input before stopping the containers.
+Windows:
```

Comment:
While the commands below don't need Git-Bash like done below I think it should suggest it all the same. Someone following the instructions will have to open a new shell just to be able to properly close it.